www/src/utils/node/docs.js

```diff
@@ -234,9 +248,10 @@ exports.onCreateNode = ({ node, actions, getNode }) => {
     if (fileNode.sourceInstanceName === `docs`) {
       slug = docSlugFromPath(parsedFilePath)
       locale = "en"
+      section = slug.split("/")[1]
 
       // Set released status and `published at` for blog posts.
-      if (_.includes(parsedFilePath.dir, `blog`)) {
```

Comment:
before, this was giving us false positives like "tutorial/blog-netlify-cms-tutorial/"